elytron/src/main/java/org/wildfly/extension/elytron/AggregateRealmDefinition.java

```diff
@@ -122,18 +131,26 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
             final List<InjectedValue<SecurityRealm>> authorizationRealmValues = new ArrayList<>();
             ModelNode authorizationRealmNode = AUTHORIZATION_REALM.resolveModelAttribute(context, model);
 
+            String principalTransformer = PRINCIPAL_TRANSFORMER.resolveModelAttribute(context, model).asStringOrNull();
+            final InjectedValue<PrincipalTransformer> principalTransformerValue = new InjectedValue<PrincipalTransformer>();
```

Comment:
There should be a null check here to make sure these three lines only get run if ```principalTransformer``` is not null.

Reply:
Fixed